src/components/file-dropzone/examples/file-dropzone.tsx

```diff
@@ -0,0 +1,69 @@
+import { FileInfo } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Basic example
+ */
+@Component({
+    tag: 'limel-example-file-dropzone',
+    shadow: true,
+    styleUrl: 'file-dropzone.scss',
+})
+export class FileDropzoneExample {
+    @State()
+    private disabled = false;
+
+    @State()
+    private readonly = false;
+
+    @State()
+    private files: FileInfo[] = [];
+
+    public render() {
+        return [
+            <limel-file-dropzone
+                onFilesSelected={this.handleDrop}
+                disabled={this.disabled || this.readonly}
```

Comment:
you have to write all of the required props in the first example. So the `text` prop or whatever we end up calling it should be here, saying "Drop your files here"

Reply:
The text is not a required prop.

Reply:
it should be 🤔 if it's not, add a `?` in front of it then 😅

Reply:
I feel it has to be a required pro, otherwise _nobody_ will understand why the border appeared there and what is going on (when they drag some file).

Reply:
text already with `?`, but the icon is the required one, do you think we need to have it opposite? Like icon optional and text required?

Reply:
I don't think it's hard to understand without a text tbh,  😅 especially when we have an Icon 

https://github.com/Lundalogik/lime-elements/assets/50618208/dac9f625-e569-449b-9338-892686571caf

Reply:
Alright, I made the text required 😅
Let's be safe 😊